src/asciiRegex.js

```diff
@@ -10,7 +10,7 @@ const names = flatten(
   })
 ).join("|");
 
-const edgeCases = ["http", "https"].join("|");
+const edgeCases = "https?:\\S*";
```

Comment:
We want to keep this an array and support both http and https URLs.

Reply:
Since the `?` in `https?` makes the `s` optional, it does support both http and https URLs.

Reply:
Oh, definitely 🤦